integration-commons/src/main/java/org/symphonyoss/integration/BaseIntegration.java

```diff
@@ -125,16 +126,20 @@ public void registerUser(String integrationUser) {
 
     Application application = properties.getApplication(integrationUser);
 
+    if (application == null) {
+      throw new CertificateNotFoundException("Certificate file unknown");
```

Comment:
@dnsymphony I changed it to provide a fallback using the component name as the default certificate name. It seems more reasonable than throws an exception.

In theory, this scenario should never happens because the bootstrap process evaluates if the application is configured properly in the YAML file.